src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -121,4 +142,8 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws ConduitAPIException, IOException {
         return conduit.perform(methodName, params);
     }
+
+    protected JSONObject callConduit(String methodName, JSONObject params) throws ConduitAPIException, IOException {
```

Comment:
Ah! This review would be easier using phabricator. ;-)

See commit message of 1fb2bdc3f9c187495d91054b56b5acc32d09c2b9  The params to `harbormaster.createartifact` are a nested json object so it can't be `Map<String,String>`.  I think `Map<Object,Object>` might work but is kind of icky and doesn't have a many convenience methods as `JSONObject`.